### DISCUSSION TITLE
Hide pupil UIs improvement

### DISCUSSION THREAD

**Discussion by fneitzel** [pupil-labs/pupil#2277] [Ideas]
I always liked that i am able to hide the settings menu. In version 1.x this is not available. 
For some usecases it would be nice to hide the status panel and control buttons on the left side as well.
Please make it possible to hide all menus - so just the gazedot is visible in the world cam.

**Comment by papr** [comment]
You can click on the icon on the right twice to hide the menu.

**Comment by fneitzel** [comment]
Ok. I mean the icon menu!

**Comment by papr** [comment]
The icon menu does not overlap the image. There is no need to hide it to see the full frame.

**Comment by papr** [comment]
Hiding the thumb icons is an idea that we have thought about as well. It was not a priority though.

**Comment by fneitzel** [comment]
Hiding the thumb icons and other stats for screen grabbings/ screen recording purposes of pupil capture would be a nice to have for the future.

**Comment by cpicanco** [comment]
Yes, specially when you need "Natural features calibration".
